src/App.test.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  const renderApp = () => {
+    const queries = render(
+      (<App />),
+    );
```

Comment:
```suggestion
    const queries = render((
      <App />
    ));
```

이렇게 작성하시면 `,` 도 없앨 수 있고 보기에도 더 좋은 것 같은데 비교했을 때 어떤게 더 나은 것 같나요?